pkgs/tools/networking/corerad/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "corerad";
+  version = "0.1.2";
+
+  goPackagePath = "github.com/mdlayher/corerad";
+
+  src = fetchFromGitHub {
+    owner = "mdlayher";
+    repo = "corerad";
+    rev = "v${version}";
+    sha256 = "11aab57xg3m6g669jqw3h6gwfr43i33xhx9jc0jw31hx335ddghs";
+  };
+
+  modSha256 = "0cfjsiiiff0k1v5j837s4jbp02snqrmv08grm69b3wjhr4145886";
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/mdlayher/corerad";
+    description = "CoreRAD extensible and observable IPv6 NDP RA daemon";
+    license = licenses.asl20;
+    maintainers = with maintainers; [ mdlayher ];
```

Comment:
```suggestion
    maintainers = with maintainers; [ mdlayher ];
    platforms = platforms.linux;
```
to avoid:
```
internal/corerad/metrics.go:151:16: undefined: getIPv6Autoconf
builder for '/nix/store/nxwf3jpcrm5a295k7886xnmccvkq4kys-corerad-0.1.2.drv' failed with exit code 1
error: build of '/nix/store/nxwf3jpcrm5a295k7886xnmccvkq4kys-corerad-0.1.2.drv' failed
```